htdocs/salaries/card.php

```diff
@@ -226,7 +226,8 @@
 			$paiement->amounts      = array($object->id=>$amount); // Tableau de montant
 			$paiement->paiementtype = $type_payment;
 			$paiement->num_payment  = GETPOST("num_payment", 'alphanohtml');
-			$paiement->note = GETPOST("note", 'none');
+			$paiement->note			= GETPOST("note", 'none');
+			$paiement->fk_user		= GETPOST("fk_user", 'int') > 0 ? GETPOST("fk_user", "int") : 0;
```

Comment:
There is no property fk_user on a payment of salary. Payment of salary is done on a salary and the user paid is fk_user on salary object.

Reply:
I'm agree that fk_user has moved to llx_salary in v14 but I needed fk_user for $paiement->addPaymentToBank() (line 242) to give employee id in bank_url.

At the base, with type salary, it's doesn't work too...

Reply:
If you have the payment (payment id of full object), you can guess the salary so the employee.
Note: setting non existing properties is nonmore allowed in php8.